SpaceGame/Assets/FuelGaugeRotator.cs

```diff
@@ -4,20 +4,27 @@
 
 public class FuelGaugeRotator : MonoBehaviour
 {
+    //fill factor
     private float amount = 1;
-
-    [SerializeField] private Timer timer;
-
-
+    //radial slider image reference
+    [SerializeField] private Image m_Slider = null;
+    //fill value for radial fill
+    private float m_radialFillAmount = 0.5f;
     public void SetPercentage(float percentage)
     {
         amount = percentage;
         if (amount < 0) amount = 0;
         if (amount > 1) amount = 1;
+        //fill is halved for radial fill -> fill for circle: max is 0.5f min is 0
+        m_radialFillAmount = amount * 0.5f;
```

Comment:
no, this is ridiculously overparamterized, set the fillAmount to `amount/2F` instead, I do not care, that is is one millionth of a nanoseconds slower.